docs/models/openai.md

```diff
@@ -309,6 +309,50 @@ agent = Agent(model)
 ...
 ```
 
+### DashScope
+
+To use Qwen models via [Alibaba Cloud DashScope](https://www.alibabacloud.com/en/product/modelstudio), you can set the `DASHSCOPE_API_KEY` environment variable and use [`DashScopeProvider`][pydantic_ai.providers.dashscope.DashScopeProvider] by name:
```

Comment:
Is `DashScopeProvider` really the most appropriate/recognizable name? I see DashScope mentioned only once on https://www.alibabacloud.com/en/product/modelstudio, so maybe it should just be `AlibabaProvider`?